librz/core/cmd/cmd_analysis.c

```diff
@@ -372,41 +360,21 @@ static bool core_analysis_name_print(RzCore *core, RzCmdStateOutput *state) {
 	return true;
 }
 
-static void print_trampolines(RzCore *core, ut64 a, ut64 b, size_t element_size) {
-	int i;
-	for (i = 0; i < core->blocksize; i += element_size) {
-		ut32 n;
-		memcpy(&n, core->block + i, sizeof(ut32));
-		if (n >= a && n <= b) {
-			if (element_size == 4) {
-				rz_cons_printf("f trampoline.%x @ 0x%" PFMT64x "\n", n, core->offset + i);
-			} else {
-				rz_cons_printf("f trampoline.%" PFMT32x " @ 0x%" PFMT64x "\n", n, core->offset + i);
-			}
-			rz_cons_printf("Cd %zu @ 0x%" PFMT64x ":%zu\n", element_size, core->offset + i, element_size);
-			// TODO: add data xrefs
+static void print_trampolines(RzCore *core, ut64 minimum, ut64 maximum,
+		size_t element_size) {
+	bool big_endian = rz_config_get_b(core->config, "cfg.big_endian");
+	for (int i = 0; i < core->blocksize; i += element_size) {
+		ut32 n = rz_read_ble32(core->block + i, big_endian);
+		if (n < minimum || n > maximum) {
+			continue;
 		}
-	}
-}
-
-static void cmd_analysis_trampoline(RzCore *core, const char *input) {
-	int bits = rz_config_get_i(core->config, "asm.bits");
-	char *p, *inp = strdup(input);
-	p = strchr(inp, ' ');
-	if (p) {
-		*p = 0;
-	}
-	ut64 a = rz_num_math(core->num, inp);
-	ut64 b = p ? rz_num_math(core->num, p + 1) : 0;
-	free(inp);
-
-	switch (bits) {
-	case 32:
-		print_trampolines(core, a, b, 4);
-		break;
-	case 64:
-		print_trampolines(core, a, b, 8);
-		break;
+		if (element_size == 4) {
+			rz_cons_printf("f trampoline.%x @ 0x%" PFMT64x "\n", n, core->offset + i);
+		} else {
+			rz_cons_printf("f trampoline.%" PFMT32x " @ 0x%" PFMT64x "\n", n, core->offset + i);
```

Comment:
i'm not sure there is much difference between `%x` and `PFMT32x` since `n` is always `ut32`

Reply:
Good point, kept `PFMT32x` for clarity and removed `%x`